librz/util/unum.c

```diff
@@ -32,19 +31,25 @@ static void rz_srand(int seed) {
 #endif
 }
 
-static int rz_rand(int mod) {
+static ut64 rz_rand(ut64 mod) {
+	ut64 r;
 #if HAVE_ARC4RANDOM_UNIFORM
-	return (int)arc4random_uniform(mod);
+	r = (ut64)arc4random_uniform(mod);
```

Comment:
See also https://stackoverflow.com/questions/26549830/swift-random-number-for-64-bit-integers